### PR TITLE
Add verify_end_to_end_purchase.py

### PR DESCRIPTION
**Pull request by jasnoludek**

### CODE REVIEW

File: Tests/verify_end_to_end_purchase.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
+import time
+import unittest
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service as ChromeService
+from webdriver_manager.chrome import ChromeDriverManager
+
+from Pages.home_page import HomePage
```
Comment (techcoachralph) [inline Tests/verify_end_to_end_purchase.py:7]: I would put Pages in lower case as it is a package. `pages.home_page`

File: Tests/verify_end_to_end_purchase.py [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+        # Login to account
+        homepage = HomePage(browser)
+        homepage.click_myAccount()
+
+        login = LoginPage(browser)
+        login.enter_username('shopper@gmail.com')
+        login.enter_password('MyPassword1234$')
+        login.click_login()
+
+        time.sleep(2)
```
Comment (techcoachralph) [inline Tests/verify_end_to_end_purchase.py:36]: I would recommend using a WebDriverWait function here. It's not good to use time.sleep when automating

File: Tests/verify_end_to_end_purchase.py [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+
+        # Navigate back to home page
+        myaccount = MyAccountPage(browser)
```
Comment (techcoachralph) [inline Tests/verify_end_to_end_purchase.py:39]: Recommendation to group instantiating page objects at the top of the test so you don't have to do them through the test. It is cleaner and easier to read

**Review by techcoachralph — CHANGES_REQUESTED** [review]
This was a GREAT job! The only serious feedback would be to change the time.sleep to a WebDriver wait function.

File: Tests/verify_end_to_end_purchase.py (hunk 1 above)
Comment (jasnoludek) [reply Tests/verify_end_to_end_purchase.py]: Pages package renamed to pages per naming conventions.

File: Tests/verify_end_to_end_purchase.py (hunk 2 above)
Comment (jasnoludek) [reply Tests/verify_end_to_end_purchase.py]: WebDriverWait replaces time.sleep where necessary.

File: Tests/verify_end_to_end_purchase.py (hunk 3 above)
Comment (jasnoludek) [reply Tests/verify_end_to_end_purchase.py]: Page objects now instantiated at top of the test.

File: Locators/locators.py [hunk 4]
```diff
@@ -97,6 +97,6 @@ class Locators():
     place_order_button_id = 'place_order'
 
     # Order received confirmation
-    order_received_confirmation_css = '//*[@id="post-8"]/div/div/div/p'
+    order_received_confirmation_xpath = '//*[@id="post-8"]/div/div/div/p'
```
Comment (techcoachralph) [inline Locators/locators.py:100]: this is not the safest xpath. It looks like it can break easily

**Review by techcoachralph — APPROVED** [review]

**Pull request merged**